docs/tool/index.md

```diff
@@ -21,6 +21,7 @@
 |[2LD](../kh2/file/type/2ld.md#layout) | [OpenKh.Tools.LayoutEditor](./LayoutEditor/index.md)
 |[2DD](../kh2/file/type/2ld.md#sequence) | [OpenKh.Tools.LayoutEditor](./LayoutEditor/index.md)
 |[DOCT](../kh2/file/type/doct) | [OpenKh.Command.DoctChanger](./DoctChanger/index.md)
+|[COCT](../kh2/file/type/coct) | [OpenKh.Command.CoctChanger](./CoctChanger/index.md)
```

Comment:
Can you add .md to the DOCT and COCT link?

Reply:
Thanks I'll fix it.